src/domain/IndexPage/IndexPage.module.scss

```diff
@@ -3,6 +3,9 @@
 
 $koros-margin: -5rem;
 
+.pageContainer {
+  background-color: white;
+}
 .korosContainer {
```

Comment:
Remove the koros related styling from here as they are moved to the Hero component.